test/Unit/Task/FileSizeTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHPTest\Unit\Task;
+
+use GrumPHP\Collection\FilesCollection;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use GrumPHP\Task\FileSize;
+use GrumPHP\Task\TaskInterface;
+use Prophecy\Prophecy\ObjectProphecy;
+use Symfony\Component\Finder\SplFileInfo;
+
+class FileSizeTest extends AbstractTaskTestCase
```

Comment:
FYI : moved the `AbstractTaskTestCase` into src so that it can be used by extensions.